opus/application/static_media/js/browse.js

```diff
@@ -599,12 +599,14 @@ var o_browse = {
         // trigger an artificial DOMMouseScroll event with delta -650
         //$( ".gallery-contents" ).trigger( e );
```

Comment:
what are we going to do w/this bit of comment?

Reply:
This is the old comment, I'm not quite sure about what that TODO comment is. Maybe we can remove that? (Because now setScrollbarOnSlide is setting the scrollbar position for both gallery and table view)